README.md

```diff
@@ -1,3 +1,77 @@
-# tensorbay-python-sdk
+# TensorBay Python SDK
 
-Graviti TensorBay Python SDK
+TensorBay Python SDK is a python library to access TensorBay and manage your datasets.  
+It provides:
+
+-   A pythonic way to access your TensorBay resources by TensorBay OpenAPI.
+-   An easy-to-use CLI tool `gas` (Graviti AI service) to communicate with TensorBay.
+-   A consistent dataset format to read and write your datasets.
+
+## Installation
+
+```console
+pip3 install tensorbay
+```
+
+## Usage
+
+An **AccessKey** is needed to communicate with TensorBay.
+Please visit [this page](https://gas.graviti.cn/access-key) to get an **AccessKey** first.
+
+### Authorize a client object
+
+```python
+from tensorbay import GAS
+gas = GAS("<YOUR_ACCESSKEY>")
+```
+
+### Create a Dataset
```

Comment:
The title is not very accurate because a `dataset client`, instead of a `dataset`, is created in the code below.